components/Common/Pagination/index.module.css

```diff
@@ -0,0 +1,65 @@
+.pagination {
+  @apply grid
+    items-center
+    justify-between
+    gap-y-5
+    [grid-template-areas:'page-buttons_page-buttons_page-buttons''previous-button_._next-button']
+    md:gap-y-0
+    md:[grid-template-areas:'previous-button_page-buttons_next-button'];
+}
+
+.previousButton,
+.nextButton {
+  @apply flex
+    items-center
+    gap-2
+    text-sm;
+}
+
+.previousButton {
+  @apply [grid-area:previous-button];
+}
+
+.nextButton {
+  @apply [grid-area:next-button];
+}
+
+.arrowIcon {
+  @apply h-5
+    shrink-0
+    text-neutral-600
+    dark:text-neutral-400;
+}
+
+.list {
+  @apply flex
+    list-none
+    justify-center
+    gap-0.5
+    [grid-area:page-buttons];
+}
+
+.listItem {
+  @apply flex
+  h-10
+    w-10
+    items-center
+    justify-center
+    rounded
+    px-3
+    py-2.5
+    !text-neutral-800
```

Comment:
I'm unfamiliar with the negate statement here. What is it supposed to do 👀

Reply:
It adds `!important` to the declaration, which is something I avoid at all costs. Nonetheless, the thing is that anchor links are being applied some styles in the `base.css` file, and I thought that modifying those could have a negative impact in other areas that are counting with those base styles, so preferred to use `!` instead of modifying those:

<img width="474" alt="image" src="https://github.com/nodejs/nodejs.org/assets/93147810/fce16071-38a1-471a-9ab6-9125f72db93e">

Reply:
You probably want to do:

```css
.listItem a {
  @apply text-neutral-800
}
```

Probably instead of `!important`

Reply:
This did not work. The thing is that `.listItem` is actually being set on the anchor already, so unless we modify those base styles, we will need to keep important.